pom.xml

```diff
@@ -47,10 +47,31 @@
     </pluginRepositories>
 
     <properties>
+        <axis2.transport.version.1.1.0v15>1.1.0-wso2v15</axis2.transport.version.1.1.0v15>
+        <carbon.apimgt.version.2.0.0>2.0.0-SNAPSHOT</carbon.apimgt.version.2.0.0>
```

Comment:
PR is put on hold till carbon apimgt 2.0.0 is released and the version is updated here.